src/components/Header/Header.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { connect } from 'react-redux';
 // import { Link } from 'react-router-dom';
 import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
```

Comment:
@evgenyboxer you can also use `withTranslation` with a function component.

`export default withTranslation()(Header)` then you'll have `t` available in the Header props directly.

Reply:
yeah... but since we are using hooks, shouldn't we use them? to me its just easier to use them on functional components and the higher order component on classes.

Reply:
Yeah I'm not fussed to be honest. I would use `withTranslation` to remove that extra line in the function but that's probably my OCD talking.